### PR TITLE
Update howto

### PR DESCRIPTION
**Pull request by i386x**
Update info how to deploy and run test harness on Open Shift.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,14 +118,20 @@ add the `ServiceAccount`, `ImageStream`, and `DeploymentConfig` from
     oc new-project linux-system-roles-test
```
Comment (i386x) [inline README.md]: Shouldn't it be `oc new-project linux-system-roles`?

File: README.md [hunk 2]
```diff
@@ -118,14 +118,20 @@ add the `ServiceAccount`, `ImageStream`, and `DeploymentConfig` from
...
     oc create -f openshift-objects.yml
 
-The service account needs to be in the privileged scc. Edit it with
+The service account needs to be in the privileged scc. If you have appropriate
+permissions, you can edit it with
 
     oc edit scc privileged
```
Comment (i386x) [inline README.md]: I think just `oc edit scc ci-pipeline-scc` should be enough.

File: README.md [hunk 3]
```diff
@@ -118,14 +118,20 @@ add the `ServiceAccount`, `ImageStream`, and `DeploymentConfig` from
...
 
 and add `- system:serviceaccount:linux-system-roles:tester` under `users`.
 
+If you need to run container as root (for example for accessing `/dev/kvm`),
+apply following file:
+
+    oc replace -f openshift-objects-root.yml
```
Comment (i386x) [inline README.md]: Alternatively, we can use `oc patch` command:
```sh
oc patch dc linux-system-roles --type json -p '[{ "op": "add", "path": "/spec/template/spec/securityContext/runAsUser", "value": "0" },{ "op": "remove", "path": "/spec/template/spec/containers/0/securityContext/fsGroup" },{ "op": "add", "path": "/spec/template/spec/volumes/0/secret/defaultMode", "value": "0400" },{ "op": "add", "path": "/spec/template/spec/volumes/1/configMap/defaultMode", "value": "0644" }]'
```
More info [here](https://labs.consol.de/development/2019/04/08/oc-patch-unleashed.html) and [here](http://erosb.github.io/post/json-patch-vs-merge-patch/).

**Comment by pcahyna** [comment]
apparently there is no information on how to create two deployments in the same project: production and staging, which is what we are doing.

**Comment by yontalcar** [comment]
@pcahyna I see it there

**Pull request merged**